discovery-infra/start_discovery.py

```diff
@@ -353,6 +431,23 @@ def nodes_flow(client, cluster_name, cluster, image_path):
                 utils.config_etc_hosts(cluster_info)
                 utils.wait_for_cvo_available()
 
+def _get_libvirt_nodes_from_tf_state(network_name, tf_state):
+    nodes = _extract_nodes(tf_state, network_name, consts.NodeRoles.MASTER)
+    nodes.update(_extract_nodes(tf_state, network_name, consts.NodeRoles.WORKER))
+    return nodes
+
+def _extract_nodes(tf_state, network_name, role):
```

Comment:
Maybe call it extract_nodes_from tfstate  ?  Maybe put in util?

Reply:
It's a workaround, I didn't want to put it in utils because it will be "public".